sdk/keyvault/azkeys/client.go

```diff
@@ -558,16 +536,11 @@ func (p *PurgeDeletedKeyOptions) toGenerated() *generated.KeyVaultClientPurgeDel
 }
 
 // PurgeDeletedKeyResponse contains the response from method Client.PurgeDeletedKey.
-type PurgeDeletedKeyResponse struct {
-	// RawResponse contains the underlying HTTP response.
-	RawResponse *http.Response
-}
+type PurgeDeletedKeyResponse struct{}
```

Comment:
Should have a comment like [this](https://github.com/Azure/autorest.go/blob/track2/test/autorest/complexgroup/zz_generated_response_types.go#L28).